CHANGES.md

```diff
@@ -1,3 +1,13 @@
+## edge-19.9.5
+
+This edge release is a release candidate for `stable-2.6`.
+
+* CLI
+  * Added headers to the JSON output for `linkerd tap`
+* Controller
+  * Added support for a node selector in the Helm chart, so users can choose
```

Comment:
I think it will be clearer with, _Added node selector constraints, so users can control which nodes the control plane is deployed to_.